pkgs/development/libraries/libsecret/default.nix

```diff
@@ -1,15 +1,23 @@
-{ stdenv, fetchurl, glib, pkgconfig, gettext, libxslt, python3, docbook_xsl, docbook_xml_dtd_42
+{ stdenv, fetchurl, fetchpatch, glib, pkgconfig, gettext, libxslt, python3, docbook_xsl, docbook_xml_dtd_42
```

Comment:
Maybe getting a bit long? Rewrap?

Reply:
I believe this is what is being suggested https://github.com/NixOS/nixpkgs/blob/1177bce848ae759a1fcc9afc2e135adaf41a092a/pkgs/desktops/pantheon/granite/default.nix#L1

Reply:
By whom? Not me!

Reply:
@alyssais I meant that is what I believe what is being represented by "Maybe getting a bit long? Rewrap?". I also make a similar request to @dtzWill frequently with their PRs.

Reply:
No, I’m not for one line per argument at all! But a reasonable length limit, like 80, is good for readability.